Kudu.SiteManagement/Configuration/IKuduConfiguration.cs

```diff
@@ -79,6 +80,21 @@ public string ApplicationsPath
             }
         }
 
+        private const string DefaultIisConfigurationFile = "%windir%\\system32\\inetsrv\\config\\applicationHost.config";
+
+        public string IisConfigurationFile
+        {
+            get
+            {
+                if(_section == null || _section.IisConfigurationFile == null)
+                    return Environment.ExpandEnvironmentVariables(DefaultIisConfigurationFile);
+
```

Comment:
Slightly confused on this one as the rest of this file uses the same style (I tried to match it)